dart/lib/src/protocol/mechanism.dart

```diff
@@ -15,15 +15,15 @@ class Mechanism {
   final String type;
 
   /// Optional human readable description of the error mechanism and a possible hint on how to solve this error
-  final String description;
+  final String? description;
 
   /// Optional fully qualified URL to an online help resource, possible interpolated with error parameters
-  final String helpLink;
+  final String? helpLink;
 
   /// Optional flag indicating whether the exception has been handled by the user (e.g. via try..catch)
-  final bool handled;
+  final bool? handled;
```

Comment:
Should this be non-nullable?

Reply:
nullable, https://develop.sentry.dev/sdk/event-payloads/exception/#attributes-1
it's optional and false by default, no need to serialize a value that is false-by-default